pkg/utils/net-attach-def.go

```diff
@@ -207,6 +207,12 @@ func CreateNetworkStatuses(r cnitypes.Result, networkName string, defaultNetwork
 		}
 	}
 
+	var defaultNetworkStatus *v1.NetworkStatus
+	if len(networkStatuses) > 0 {
+		// Set the default network status to the last network status.
+		defaultNetworkStatus = networkStatuses[len(networkStatuses)-1]
```

Comment:
in https://github.com/k8snetworkplumbingwg/network-attachment-definition-client/issues/74#issuecomment-2587640738 we decided to (try to ...):
- lastInfIdx could try to find the last sandbox interface
- if there isn't any sandbox interface, default to the last interface

I assumed we'd try to point to the last interface that featured a sandbox defined, and if we couldn't find one, we'd just grab the latest.

I feel I'm missing something ...